alacritty/src/config/mod.rs

```diff
@@ -204,6 +204,14 @@ fn print_deprecation_warnings(config: &Config) {
              lines"
         );
     }
+
+    if config.scrolling.auto_scroll.is_some() {
+        warn!(
+            target: LOG_TARGET_CONFIG,
+            "Config scrolling.auto_scroll was removed as well as the corresponding feature, \
+             there's no need to keep it anymore"
```

Comment:
`"Config scrolling.autoscroll and its corresponding feature have been removed"`?

Additionally, do we really need to say "corresponding feature"? Just sounds weird to me. If we have to keep it, maybe change it to something like "related functionality"? (would also need to be changed in the changelog)

Reply:
Personally, I'd go for short and sweet. I think people would understand a warning for just "Config scrolling.auto_scroll has been removed".

Reply:
If the config option has been removed, it's not immediately clear if it is now default `true` or default `false` though.

But I had the same thought as you, I'm not a huge fan of the naming, but I don't have any better proposal.

Reply:
"Config scrolling.auto_scroll has been removed, and is now always enabled"?

Reply:
I like that wording. Though I would still prefer if the changelog said something more along the lines of `related functionality` than `corresponding feature`. My contrived reasoning is that it could be confused with an actual `#[cfg(feature = "autoscroll")]`.

Reply:
We don't have such a feature and our changelogs are not geared torwards Rust users. So I don't think this is very relevant.

Though I do think that `functionality` is probably the better term in general.